app/controllers/repository_columns_controller.rb

```diff
@@ -34,6 +34,18 @@ def create
     raise NotImplementedError
   end
 
+  def describe_all
+    response_json = @repository.repository_columns.map do |column|
```

Comment:
Can we also merge default columns here, it will be the full list of columns.  But not sure which `id` default columns will have.